apps/ledger-live-mobile/src/families/elrond/components/Body/components/Drawer/Drawer.tsx

```diff
@@ -45,15 +45,15 @@ const Drawer = (props: DrawerPropsType) => {
       actions={actions}
       account={account}
       data={items}
-      ValidatorImage={({ size }) => (
```

Comment:
I prefer the {{ size }} writing than image.size

Reply:
Changed. 👍